core/repositories/PostRepository.php

```diff
@@ -89,4 +89,18 @@ public function getById(int $id)
         $post = current($this->getAll('', $id));
         return $post;
     }
+
+    public function deleteById($id)
+    {
+        $sql = "DELETE from `posts`
+        WHERE `id` = :id
+        LIMIT 1";
+
+
+        $stmt = $this->getDB()->prepare();
+        $stmt->bindValue(':id', $id);
+        $stmt->execute();
+
+        return $this->getDB();
```

Comment:
I believe you should return true to indicate to controller that deletion is done properly